example/src/assets/data/playlist.json

```diff
@@ -35,8 +35,9 @@
   },
   {
     "url": "https://ais-sa5.cdnstream1.com/b75154_128mp3",
-    "title": "Smooth Jazz 24/7 (Live Radio)",
+    "title": "Smooth Jazz 24/7",
     "artist": "New York, NY",
-    "artwork": "https://react-native-track-player.js.org/example/smooth-jazz-24-7.jpeg"
+    "artwork": "https://react-native-track-player.js.org/example/smooth-jazz-24-7.jpeg",
+    "isLiveStream": true
   }
-]
+]
```

Comment:
Technically a stylistic thing and subject to preference. But you should always have [newlines at the end of files](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).

Reply:
This is so odd - prettier seems to be removing the new line for some reason... Anyway, added it in there now